Musicplayer.css

```diff
@@ -1,64 +1,38 @@
+/* Import  font from google fonts*/
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&display=swap');
+
 body{
 	margin:0;
 	padding:0;
 	background:white;
-	overflow: scroll;
-	overflow-x: hidden;	
+	
 }
 
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 10% 75%;
-	grid-template-rows: auto auto auto auto auto;
+	grid-template-columns: 15% 35% 50%;
+	grid-template-rows: 58px auto auto auto auto;
 	height: 100vh;
 }
 
-.img_1{
-	grid-column-start: 1;
-	grid-column-end: span 4;
-	height:300px;
-	background:red;
-}
-
-.img_1 img{
-	display:block;
-	margin-left:auto;
-	margin-right:auto;
-	height:300px;
-	width:100%;
-}
-
-.footer{
-	background-color: white;
-	grid-column-end: span 4;
-	padding-top:25px;
-	text-align: center;
-	color:black;
-	height:50px;
-}
-
 .navigation{
 	box-sizing: border-box;
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	margin-top:-10px;
```

Comment:
21/9: Unlike layout one, in layout two I decided to make the apply the navbar at the top so I decided to remove of of the setting in the navigation div box. I will setting the color latter in the nav.